ubuntu-2004.json

```diff
@@ -10,14 +10,17 @@
       "iso_checksum": "caf3fd69c77c439f162e2ba6040e9c320c4ff0d69aad1340a514319a9264df9f",
       "iso_checksum_type": "sha256",
 
-      "ssh_username": "ubuntu",
-      "ssh_password": "ubuntu",
-      "ssh_handshake_attempts": "20",
+      "ssh_username": "{{user `ssh_user`}}",
+      "ssh_password": "{{user `ssh_pass`}}",
+      "ssh_handshake_attempts": "30",
```

Comment:
This is of course just an arbitrary number I found worked reasonably well, but were you finding it was often hitting more than 20 failures?